Spatial_Engine/Query/Centroid.cs

```diff
@@ -32,6 +32,19 @@ namespace BH.Engine.Spatial
 {
     public static partial class Query
     {
+        /******************************************/
+        /****            IElement0D            ****/
+        /******************************************/
+        
+        [Description("Queries the centre of weight of an IElement0D. Always returns the point location due to zero-dimensionality of an IElement0D.")]
+        [Input("element0D", "The IElement0D with the geometry to get the centre of weight of.")]
+        [Output("centroid", "The Point at the centre of weight for the homogeneous geometrical representation of the IElement0D.")]
+        public static Point Centroid(this IElement0D element0D)
```

Comment:
```suggestion
        public static Point Centroid(this IElement0D element0D, double tolerance = Tolerance.Distance)
```
Tolerance is needed to be picked up by the interface.
Similar issue is in the ControlPoints method, where `IElement2D` have different arguments then others, but that might be out of scope.

Reply:
Nice catch 👍